rust-runtime/aws-smithy-http/src/byte_stream/bytestream_util.rs

```diff
@@ -8,13 +8,19 @@ use futures_core::{ready, Stream};
 use http::HeaderMap;
 use http_body::{Body, SizeHint};
 use std::future::Future;
-use std::path::{Path, PathBuf};
+use std::path::PathBuf;
 use std::pin::Pin;
 use std::task::{Context, Poll};
 use tokio::fs::File;
 use tokio::io;
 use tokio_util::io::ReaderStream;
 
+use crate::body::SdkBody;
+
+use super::{ByteStream, Error};
+
+const DEFAULT_BUFFER_SIZE: usize = 4096;
```

Comment:
Could you write a comment about how this buffer size was chosen?

Reply:
Done